app/static/css/themes/nord-light.css

```diff
@@ -0,0 +1,52 @@
+html {
+    background-color: #D8DEE9b3 !important;
+}
+
+body {
+    background-color: #f9fafb !important;
```

Comment:
How come the `body` element color is different from the `html` one? It should probably just be one or the other. From a personal standpoint, I think `D8DEE9b3` looks better, although you could probably keep it just straight rgb as `D8DEE9` and leave off the alpha.